modules/veteran_verification/app/controllers/veteran_verification/application_controller.rb

```diff
@@ -11,6 +12,21 @@ class ApplicationController < ::OpenidApplicationController
     # so we set that default.
     before_action { set_default_format_to_json }
 
+    def authenticate_token
+      return false if token.blank?
+
+      @session = Session.find(token)
+      establish_session if @session.nil?
+      return false if @session.nil?
+
+      open_id = if Settings.vet_verification.mock_emis
```

Comment:
Just a reminder, but knowing this is still in Draft stages.

Will need config settings to similar to mock_bgs.

https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/dev-settings.local.yml.j2#L437

Reply:
https://github.com/department-of-veterans-affairs/devops/pull/7992